packages/fiber/src/native.tsx

```diff
@@ -16,6 +16,7 @@ export type { ThreeEvent, Events, EventManager, ComputeFunction } from './core/e
 export { createEvents } from './core/events'
 export type { ObjectMap, Camera } from './core/utils'
 export * from './native/Canvas'
+export * from './native/polyfills'
```

Comment:
It affects models too (glb in my case), but I don't know how to solve it in case of monkeypatching, i just modified workaround https://github.com/pmndrs/react-three-fiber/issues/2577#issuecomment-1316289201.

Reply:
For sure. We'll have to polyfill at the native entrypoint.